packages/cms-cli/commands/functions.js

```diff
@@ -18,6 +19,7 @@ exports.builder = yargs => {
       ...list,
       aliases: 'ls',
     })
+    .command(build)
```

Comment:
I do wonder if there's a better command name for this. Perhaps `package`, `bundle`, or `install-deps`?

Reply:
I like `package`.  Then we could have one that is `hs functions deploy` for the second step

Reply:
Renamed it to `hs functions package`